framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -525,11 +526,13 @@ private void applyDefaultsElement(final Element annotationScope, final Annotated
                 // but it doesn't work correctly and tests fail.
```

Comment:
The method applies conservative and source code defaults. I updated the comment to refer to the manual sections as the defaulting rules are too complex to explain in a javadoc comment. The method is private, so I think anyone trying to understand or change it should read those two manual sections anyway.